scripts/integration-test

```diff
@@ -85,8 +90,17 @@ trap "kill $pid_controller $pid_replica1 $pid_replica2 \
        $pid_backing_replica1 $pid_backing_replica2 \
        $pid_upgrade_replica1 $pid_upgrade_replica2" EXIT
 
+# start minio server to test s3 backup and set credential keys
+export MINIO_ACCESS_KEY=test-access-key
+export MINIO_SECRET_KEY=test-secret-key
+# create default bucket
+mkdir -p /data/backupbucket
+# run minio server
+nohup /usr/bin/minio server /data &
+
 cd integration
 find -depth -name __pycache__ -o -name "*.pyc" -exec rm -rf {} \;
 if [ -z "$NO_TEST" ]; then
+    sed -i "/[testenv] /asetenv= \n AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID} \n AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY} \n AWS_ENDPOINTS=${AWS_ENDPOINTS} \n BACKUPTARGETS=s3://backupbucket@us-east-1/backupstore,vfs:///data/backupbucket" tox.ini
```

Comment:
Use `passenv=` in the `tox.ini` instead

Reply:
`setenv` is a keyword to set environment variables into tox

Reply:
I know. I meant, it's easier to use `passenv=` because you already have those environment variables set in the shell here. Tox will simply allow those variable value to be passed to the test cases.

Compare to the `sed` here, using `passenv=` would make code easier to read and you don't need to dynamic modified the `tox.ini` in the script. Just modify the `passenv=` in the `tox.ini`.